source/security.rst

```diff
@@ -23,8 +23,19 @@ Limitations
 
 - **HTTP Traffic to Origin Servers**: Traffic to backend services, including computational resources like Jupyter servers, is currently over HTTP, which is unencrypted. Plans are underway to upgrade this to HTTPS to ensure encryption of data in transit, thereby bolstering security.
 
-Security Controls
------------------
+Controls
+^^^^^^^^
+
+These are things the the out of the box OnDemand installation will provide
+that some centers may want to change or disable altogether.
+
+- **File Access**: OnDemand lets users navigate the file system. While file permissions
+  limit what a user can view and navigate to, some centers may want to limit this even further.
+  One option is to :ref:`set-file-allowlist` to limit what directories users may navigate to.
+
+
+Additional Information
+----------------------
```

Comment:
I'm not sure about this heading. Happy to change it. Don't really know the purpose of this section, so can't really speak to it.

Reply:
sure, change it to whatever you'd like! I'll remove the monitoring and logging line.

Reply:
OK - if your'e removing those 2 maybe I can just remove the whole section and move the line about vulnerability reporting somewhere else, maybe nearer the top of the page?

Reply:
If you think that will be better please do. I was trying to put info that could be useful for security somewhere, but i'm not sure about the order. i think its a good idea for vulnerability reporting to be mentioned near the top